api/v1alpha1/hostedcluster_types.go

```diff
@@ -562,6 +572,134 @@ type IBMCloudPlatformSpec struct {
 	ProviderType configv1.IBMCloudProviderType `json:"providerType,omitempty"`
 }
 
+// PowerVSPlatformSpec defines IBMCloud PowerVS specific settings for components
+type PowerVSPlatformSpec struct {
+	// ResourceGroup is the IBMCloud Resource Group in which the cluster resides.
+	// This field is immutable. Once set, It can't be changed.
+	//
+	// +immutable
+	ResourceGroup string `json:"resourceGroup"`
```

Comment:
this fields seems to be unused by the controllers?

Reply:
This will goto ccm configuration file later.